tests/local_client_server_tcp_e2e/local_client_server_tcp_e2e.c

```diff
@@ -284,6 +304,7 @@ TEST_FUNCTION(client_and_server_connect_and_send_one_message_unsettled)
 
     // start the client
     SOCKETIO_CONFIG socketio_config = { "localhost", 5672, NULL };
```

Comment:
Ok. I can't assign port_number because this initializer only takes constant values.